lib/modules/manager/dockerfile/extract.ts

```diff
@@ -88,6 +88,7 @@ function processDepForAutoReplace(
   dep.replaceString = unfoldedLineNumbers
     .map((lineNumber) => lines[lineNumber])
     .join(linefeed);
+  dep.replaceString += linefeed;
```

Comment:
it can be any `\s` so we need to also support a space

Reply:
`lines[lineNumber]` contains the entire, untrimmed line, potentially with trailing spaces. That's why only \r?\n is added here.